### PR TITLE
Fix for #13

### LINKED ISSUE #13: Exception when having an enum in propTypes
(originally reported in https://github.com/sapegin/react-styleguidist/issues/9#issuecomment-145516722)

I can now run styleguidist, and it can display the props for one component, but when I add another one, it fails:

image

I think it's because of the enum in the other component's `propTypes`. Here are the `propTypes` of the relevant component:

```
    propTypes: {
        issue: React.PropTypes.object,
        width: React.PropTypes.oneOf([26, 40, 55, 70, 110, 155, 190, 265]),
        style: React.PropTypes.object,
    },
```

### PR DESCRIPTION
**Pull request by mik01aj**
This should fix #13.

### CODE REVIEW

File: src/components/Props/Props.js [hunk 1]
```diff
@@ -58,6 +58,9 @@ export default class Props extends Component {
 	}
 
 	renderEnum(prop) {
+		if (prop.type.value.constructor !== Array) {
```
Comment (sapegin) [inline src/components/Props/Props.js]: I think Array.isArray is better than checking a constructor.

**Comment by sapegin** [comment]
Cool, thanks!

**Comment by sapegin** [comment]
Could you please fix this and I’ll merge it?

**Comment by mik01aj** [comment]
should be ok now.

**Pull request merged**